ReferenceBackendAPI.md

```diff
@@ -0,0 +1,193 @@
+# Reference Backend API
+
+## HTTP Method
+
+The reference backend is deployed to [Twilio Functions](https://www.twilio.com/docs/runtime/functions), a serverless environment.
+
+Twilio Functions does not utilize HTTP method so you can just use `POST` for all requests.
+
+## Authentication
+
+All requests must set the `Authorization` header to the correct [passcode](README.md#deploy-the-app-to-twilio). The backend will validate the passcode before processing each request.
```

Comment:
Yeah we should mention 401 because that can be a confusing dead end.

But regarding the other errors such as expired passcode, the iOS app actually just passes it through for the UI to display just like any other fatal error. And the message that the backend provides for these cases is intuitive. So in other words I think the iOS app works well without any special handling. And I think we recommend that customers completely replace the auth system for a production app. So I imagine a customer wouldn't add custom error handling for the passcode solution. Does the web app handle it differently? I'm thinking no need to get into specific errors?

Reply:
That's a good point. I guess we don't need to get into specific errors. But, I still think that it could be useful to mention that there are two different ways to get a 401 here, and that it is also useful (if not necessary) for the UI to consume the error messages that are returned.